doc/LLGO Migration for C C++ Third-Party Libraries.md

```diff
@@ -0,0 +1,461 @@
+# LLGO Migration for C/C++ Third-Party Libraries
```

Comment:
don't use space in filename

Reply:
Modified to llgo_migration_for_c_c++_third-party_libraries